src/Entity/MollieGatewayConfig.php

```diff
@@ -222,4 +222,9 @@ public function getTranslation(?string $locale = null): MollieGatewayConfigTrans
 
         return $translation;
     }
+
+    public function hasTranslationLocale(string $locale): bool
```

Comment:
I think You send like parameter `localeCode`, maybe this will be better instead of `$locale`